src/main/java/org/gridsuite/cgmes/boundary/server/CgmesBoundaryService.java

```diff
@@ -20,15 +21,19 @@
 import java.io.UncheckedIOException;
 import java.nio.ByteBuffer;
 import java.nio.charset.StandardCharsets;
+import java.time.LocalDateTime;
+import java.util.Iterator;
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Collectors;
 
 /**
  * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ * @author Etienne Homer <etienne.homer at rte-france.com>
  */
 @Service
 class CgmesBoundaryService {
+    private static final String REGEX = "^(.*?(__ENTSOE_%sBD_).*(.xml))$";
```

Comment:
Is the character '?' needed in the regex ?

Reply:
Yes, I removed it